test/unit/models/index.test.ts

```diff
@@ -133,4 +133,38 @@ describe("test installation model", () => {
 			expect(subscription.jiraHost).toBe(renamedInstallPayload.baseUrl);
 		}
 	});
+
+	it("should return the most recent entry if there are duplicate hosts", async () => {
```

Comment:
This should be made into a describe and then separate each scenario into it's own test.  You've changed 2 functions so both needs to be tests for 0, 1 and many results.

Reply:
You're also missing the scenario of the same host/clientKey being installed and how that would work as per the scenario of the client.  You'll have to use `Installation.create` directly instead of using `install` to add those records to the DB.